src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/data/CompareUtil.java

```diff
@@ -37,10 +40,20 @@
      * @param pattern the pattern to use.
      * @param str the string to match on.
      * @return true if the string matches the pattern.
+     * @deprecated use {@link #matches(String, String, EnvVars)} instead.
      */
     boolean matches(String pattern, String str);
```

Comment:
Add a `@Deprecated` annotation as well so tools has an easier time figuring it out.